### DISCUSSION TITLE
HTML Usage Example not working

### DISCUSSION THREAD

**Discussion by Lex-Kent** [speechly/speechly#209] [Q&A] [answered]
Hi all,

I'm trying to get started with Speechly in a browser, and the simplest example I could find is not working. When I copy-paste the "Usage in HTML" example from here: https://github.com/speechly/speechly/tree/main/libraries/browser-client and replace "your-app-id" with my APP ID, I see this error in my console:

```
Uncaught Error: No AudioProcessor
    at WebsocketClient.startStream (84d30660-cb8d-4c09-9c31-2d64dd7746e0:550:23)
    at ctx.onmessage (84d30660-cb8d-4c09-9c31-2d64dd7746e0:690:33)
```

Even when I host it on a HTTPS:// server.

Can anyone explain what I've done wrong please?

Many thanks,

Lex

**Comment by langma** [comment, marked as answer]
Sorry for taking so long to answer this, but the problem is because of a problem in the example code in README.md. 

[This PR](https://github.com/speechly/speechly/pull/211/files) shows how to fix it in your version, but basically a missing `await` should fix the problem.

**Reply by Lex-Kent** [reply]
Thank you @langma but how does that help me? I'm still importing from "//unpkg.com/@speechly/browser-client?module=true".

I'm a digital artist, not a developer so I have no idea how to build this or include it locally. Is there a way I can include the version with the PR included please?

**Reply by langma** [reply]
@Lex-Kent the fix is not part of the imported code, so you need to change the code you have in your HTML. The PR is now merged, so one option is to copy-paste [the HTML example](https://github.com/speechly/speechly/tree/main/libraries/browser-client#usage-in-html) again, and it should work. You just need to add your appId to the `BrowserClient` constructor.

**Reply by Lex-Kent** [reply]
Thanks @langma, it is now working perfectly :o)
